Microsoft.Toolkit.Uwp.UI/Converters/ResourceNameToResourceStringConverter.cs

```diff
@@ -0,0 +1,54 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Reflection;
+using Windows.ApplicationModel.Resources;
+using Windows.UI.Xaml.Data;
+
+namespace TommasoScalici.UWPConvertersAndUtilities
+{
+    /// <summary>
+    /// Value converter that look up for the source string in the App Resources strings and returns its value, if found.
+    /// </summary>
+    public sealed class ResourceNameToResourceStringConverter : IValueConverter
+    {
+        private readonly ResourceLoader resourceLoader = ResourceLoader.GetForViewIndependentUse();
+
+        /// <summary>
+        /// Take the source string as a resource name that will be looked up in the App Resources.
+        /// If the resource exists, the value is returned, otherwise.
+        /// </summary>
+        /// <param name="value">The source string being passed to the target.</param>
+        /// <param name="targetType">The type of the target property, as a type reference.</param>
+        /// <param name="parameter">An optional parameter to be used to invert the converter logic.</param>
+        /// <param name="language">The language of the conversion.</param>
+        /// <returns>The string corresponding to the resource name.</returns>
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+            return resourceLoader.GetString(value?.ToString());
```

Comment:
No, it's not safe at all since the method will throw an exception if null is passed. I'll add a null check.